orc8r/cloud/deploy/terraform/orc8r-helm-aws/efs.tf

```diff
@@ -13,6 +13,8 @@
 
 # k8s requires provisioner to treat efs as a persistent volume
 resource "helm_release" "efs_provisioner" {
+  count = var.orc8r_is_staging_deployment == true ? 0 : 1
```

Comment:
To state my understanding, we don't create a new efs provisioner since we intend to use the same backend storage for the new services as well. Is that a fair understanding?

Reply:
Yes, to add more color this would be used to hold the prometheus configurations for metrics, since we do not have a prometheus in the new namespace we disabled this EFS as well.

If we enable it, the creation will fail since the efs_file_system_id would still be configured to the main orc8r value.